rllib/core/rl_trainer/trainer_runner.py

```diff
@@ -255,3 +256,8 @@ def set_state(self, state: List[Mapping[ModuleID, Mapping[str, Any]]]) -> None:
             for worker in self._workers:
                 refs.append(worker.set_state.remote(state))
             ray.get(refs)
+
+    def shutdown(self):
```

Comment:
can we modify the `__del__` method to also call shutdown? before deleting the object?

Reply:
yerp

Reply:
done